### PR TITLE
feat(UI): Improve about page

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This migrates the About page to React.

https://user-images.githubusercontent.com/12515407/224377451-259d944b-1b3a-4cae-b448-bc2048eaa1d1.mov

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

- The about page should be displayed

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Wedierre — DISMISSED** [review]

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 1]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+      description: "Platform customization configuration"
+      properties:
+        platform_name:
+          type: string
+          description: "Platform display name"
+          example: "My Custom Centreon"
+          nullable: true
+        custom_text:
+          type: string
+          description: "Custom text display on login page on JSON format"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8452]:
```suggestion
          description: "Custom text display on login page in JSON format"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 2]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+          example: |
+            '{"root": {"children": [{"detail": 0, "format": 0, "mode": "normal", "style": "",
+            "text": "Welcome on Centreon", "type": "text", "version": 1}]'
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8455]:
```suggestion
            "text": "Welcome to Centreon", "type": "text", "version": 1}]'
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 3]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+      allOf:
+        - $ref: "#/components/schemas/Configuration.Platform.Customization"
+          properties:
+            icon:
+              type: object
+              description: "Select custom icon"
+              properties:
+                id:
+                  type: number
+                  description: "Id of the icon"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8470]:
```suggestion
                  description: "ID of the icon"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 4]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+        text_position:
+          type: string
+          enum: [top, bottom]
+    Configuration.Platform.Customization.Read:
+      allOf:
+        - $ref: "#/components/schemas/Configuration.Platform.Customization"
+          properties:
+            icon:
+              type: object
+              description: "Select custom icon"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8466]:
```suggestion
              description: "Select a custom icon"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 5]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+                  description: "URL for the icon"
+                  example: "/centreon/img/media/ppm/my_custom_icon.png"
+              nullable: true
+            image:
+              type: object
+              description: "Select custom image"
+              properties:
+                id:
+                  type: number
+                  description: "Id of the image"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8491]:
```suggestion
                  description: "ID of the image"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 6]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+                  description: "URL for the image"
+                  example: "/centreon/img/media/ppm/my_custom_image.png"
+              nullable: true
+    Configuration.Platform.Customization.Write:
+      allOf:
+        - $ref: "#/components/schemas/Configuration.Platform.Customization"
+          properties:
+            icon_id:
+              type: number
+              description: "Select custom icon id"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8512]:
```suggestion
              description: "Select custom icon ID"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 7]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+              nullable: true
+            image_id:
+              type: number
+              description: "Select custom image id"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8516]:
```suggestion
              description: "Select custom image ID"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 8]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+      description: "Platform customization configuration"
+      properties:
+        platform_name:
+          type: string
+          description: "Platform display name"
+          example: "My Custom Centreon"
+          nullable: true
+        custom_text:
+          type: string
+          description: "Custom text display on login page on JSON format"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8529]:
```suggestion
          description: "Custom text display on login page in JSON format"
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 9]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+          example: |
+            '{"root": {"children": [{"detail": 0, "format": 0, "mode": "normal", "style": "",
+            "text": "Welcome on Centreon", "type": "text", "version": 1}]'
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8532]:
```suggestion
            "text": "Welcome to Centreon", "type": "text", "version": 1}]'
```

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 10]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+        text_position:
+          type: string
+          enum: [top, bottom]
+    Configuration.Platform.Image.Resize:
+      type: object
+      description: "Image resizing"
+      properties:
+        id:
+          type: number
+          description: "Id of the image to resize"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8551]:
```suggestion
          description: "ID of the image to resize"
```

File: centreon/www/front_src/src/About/Sections/Developers.tsx [hunk 11]
```diff
@@ -0,0 +1,30 @@
+import NamesList from '../NamesList';
+
+export const developers = [
+  'Adrien Gelibert',
+  'Adrien Morais',
+  'Allyriane Launois',
+  'Arnaud Buathier',
+  "Bruno D'auria",
```
Comment (cg-tw) [inline centreon/www/front_src/src/About/Sections/Developers.tsx:8]:
```suggestion
  "Bruno D'Auria",

```

File: centreon/www/front_src/src/About/translatedLabels.ts [hunk 12]
```diff
@@ -0,0 +1,14 @@
+export const labelCentreon = 'Centreon';
+export const labelDevelopedBy = 'Developed by';
+export const labelAnd = 'and';
+export const labelCommunity = 'the community';
+export const labelProjectLeaders = 'Project leaders';
+export const labelDevelopers = 'Developers';
+export const labelContributors = 'Contributors';
+export const labelSecurityAcknowledgement = 'Security acknowledgement';
+export const labelAndManyOthers = 'And many others...';
+export const labelYouCanSeeFullListOnThe = 'You can see the full list on the';
```
Comment (cg-tw) [inline centreon/www/front_src/src/About/translatedLabels.ts:10]: si le but c'est de concaténer avec "Centreon's GitHub", il faut pas mettre le "the"

File: centreon/www/front_src/src/About/Sections/Developers.tsx [hunk 13]
```diff
@@ -0,0 +1,31 @@
+import NamesList from '../NamesList';
+
+export const developers = [
+  'Adrien Gelibert',
+  'Adrien Morais',
+  'Allyriane Launois',
+  'Arnaud Buathier',
+  "Bruno D'Auria",
+
```
Comment (cg-tw) [inline centreon/www/front_src/src/About/Sections/Developers.tsx:9]:
```suggestion
```

**Review by cg-tw — DISMISSED** [review]

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 14]
```diff
@@ -8,7 +8,9 @@ info:
     + Added host groups endpoints (DELETE, POST)
     + Added host categories endpoints (DELETE, POST)
     + Added host severities endpoints (DELETE)
-    +
+    + Added platform custom configuration endpoints (GET, PUT)
+    + Added platform custom configuration for login (GET)
+    + Added image resizing endpoints (POST)
```
Comment (jeremyjaouen) [inline centreon/doc/API/centreon-api-v23.04.yaml:13]:
```suggestion
```

Comment (jeremyjaouen) [reply centreon/doc/API/centreon-api-v23.04.yaml:13]: this has been moved into the module

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 15]
```diff
@@ -4406,6 +4408,83 @@ paths:
...
+              $ref: '#/components/schemas/Configuration.Platform.Image.Resize'
+      responses:
+        '201':
+          description: "OK"
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
```
Comment (jeremyjaouen) [inline centreon/doc/API/centreon-api-v23.04.yaml:4487]:
```suggestion
```
this has been moved into the module

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 16]
```diff
@@ -8359,3 +8438,123 @@ components:
...
+          description: "ID of the image to resize"
+          example: 2
+        width:
+          type: number
+          description: "Width to resize image in px"
+          example: 30
+        height:
+          type: number
+          description: "Height to resize image in px"
+          example: 20
```
Comment (jeremyjaouen) [inline centreon/doc/API/centreon-api-v23.04.yaml:8560]:
```suggestion
```
this has been moved into the module

**Review by jeremyjaouen — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by Wedierre — APPROVED** [review]

File: centreon/doc/API/centreon-api-v23.04.yaml [hunk 17]
```diff
@@ -8358,4 +8357,4 @@ components:
           example: "centreon"
         role_id:
           type: string
-          description: "Vault role id"
+          description: "Vault role id"
```
Comment (cg-tw) [inline centreon/doc/API/centreon-api-v23.04.yaml:8360]:
```suggestion
          description: "Vault role ID"
```

**Review by cg-tw — DISMISSED** [review]

**Review by bdauria — APPROVED** [review]

**Review by SoldierBoy31 — APPROVED** [review]

**Review by Wedierre — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**